src/ui/components/form/InputBn.tsx

```diff
@@ -8,29 +8,58 @@ import { ArgComponentProps } from 'types';
 
 type Props = ArgComponentProps<BN>;
 
-export function InputBn({ onChange, typeDef }: Props): JSX.Element {
+function getMinMax(type: string): [bigint, bigint] {
```

Comment:
Not sure how exactly but maybe this type can be a bit narrowed down from the `TypeDef` stuff from polkadotjs? 

If not, I think we should better throw an Error is the type is not available in the switch case, so use of unsupported type does not go unnoticed. 🤔

Reply:
there is no unsupported type, really. it is very unlikely that this input will be used with something other than numeric types. in the rare case that happens the only problem will be that the user can't input numbers higher than `MAX_SAFE_INTEGER`